spec/index.bs

```diff
@@ -9635,19 +9635,22 @@ dictionary GPUCommandEncoderDescriptor
                 1. [$Validate the encoder state$] of |this|.
                     If it returns false, make |pass| [=invalid=] and return.
                 1. Set |this|.{{GPUCommandsMixin/[[state]]}} to "[=encoder state/locked=]".
+                1. Let |attachmentSubresources| be an empty [=list=].
+                1. For each non-`null` |colorAttachment| in |descriptor|.{{GPURenderPassDescriptor/colorAttachments}}:
+                    1. Add the [=texture subresource=] range described by
+                        |colorAttachment|.{GPURenderPassColorAttachment/view}} and, for {{GPUTextureViewDimension/"3d"}}
+                        attachments, |colorAttachment|.{{GPURenderPassColorAttachment/depthSlice}} to |attachmentSubresources|.
```

Comment:
This is a good approach!

Wording thing: Each one is not just a subresource (in 3d, a whole miplevel), but a subregion of a subresource (in 3d, one slice of one miplevel).

It also isn't a "range" - each attachment region touches exactly 1 subresource (possibly only a slice thereof).

Reply:
Tweaked the verbiage a bit, what do you think now?

Reply:
- attachmentSubresources should be renamed somehow
- clarify that "the region" that we're adding specifies both the subresource and the offset/size (not just teh offset/size)

Reply:
Hm... updated, but I'm really not convinced that I'm moving in the right direction in terms of readability. Very open to feedback on how to improve this.